### PR TITLE
CASMCMS-8393 - add console log rotation documentation.

### PR DESCRIPTION
**Pull request by dlaine-hpe**
# Description

Relates to: https://jira-pro.it.hpe.com:8443/browse/CASMCMS-8393

This adds documentation for the console services log rotation settings.

- [X] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [X] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [X] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by dlaine-hpe** [comment]
Auto-backport to release/1.5 failed due to conflict. This will have to be done manually if it is required.

**Review by jsl-hpe — APPROVED** [review]

**Comment by dlaine-hpe** [comment]
release/1.5 backport: https://github.com/Cray-HPE/docs-csm/pull/5439
release/1.4 backport: https://github.com/Cray-HPE/docs-csm/pull/5440

File: operations/conman/Configure_Log_Rotation.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+> **`NOTE`** Log rotation will move the current log file and create a new one with the original
+    location and name. If you are using a `tail` operation to watch the console log output,
+    make sure to use the `tail -F` option to automatically switch the `tail` to the new
+    file through a log rotation. Otherwise the `tail` will follow the old file which has
+    moved and is no longer being appended to with new console log information.
+
+## How log rotation works
+
+1. On a regular schedule, the log rotation will execute the following steps:
```
Comment (nrockershousen) [inline operations/conman/Configure_Log_Rotation.md]:
```suggestion
On a regular schedule, the log rotation will execute the following steps:
```
This doesn't need to be a top-level step.

File: operations/conman/Configure_Log_Rotation.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+    1. Check the size of all the current console log files.
```
Comment (nrockershousen) [inline operations/conman/Configure_Log_Rotation.md]: These steps don't need to be indented now that the top step is removed.

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**